integration_tests/src/main/python/join_test.py

```diff
@@ -119,32 +119,45 @@ def join_batch_size_test_params(*args):
                 params += [ pytest.param(obj, batch_size) ]
     return params
 
+
 @ignore_order(local=True)
 @pytest.mark.parametrize('join_type', ['Left', 'Inner', 'LeftSemi', 'LeftAnti'], ids=idfn)
 @pytest.mark.parametrize("aqe_enabled", ["true", "false"], ids=idfn)
-def test_right_broadcast_nested_loop_join_without_condition_empty(join_type, aqe_enabled):
+@pytest.mark.parametrize("join_reorder_enabled", ["true", "false"], ids=idfn)
```

Comment:
nit: and it might make the file inconsistent, but just a note that we can use native Python constants True, False as PySpark Boolean conf values

Reply:
Thanks. I have implemented this in the new test that was added since your last review.